### PR TITLE
[WIP] update dependencies

### PR DESCRIPTION
**Pull request by fanatid**
Depends: https://github.com/crypto-browserify/ripemd160/pull/14 https://github.com/crypto-browserify/sha.js/pull/36
Ref: https://github.com/crypto-browserify/createHash/issues/11 https://github.com/crypto-browserify/createHash/issues/12
- md5.js extracted to [crypto-browserify/md5.js](https://github.com/crypto-browserify/md5.js)
- ripemd160 updated for supporting streams in own repository
- sha.js updated for using [hash-base](https://github.com/crypto-browserify/hash-base) instead [cipher-base](https://github.com/crypto-browserify/cipher-base)

### CODE REVIEW

File: browser.js [hunk 1]
```diff
@@ -1,52 +1,18 @@
...
-
-  return new Hash(sha(alg))
+  alg = alg.toUpperCase()
+  switch (alg) {
+    case 'MD5':
+      return new MD5()
+    case 'RMD160':
+      return new RIPEMD160()
+    case 'RIPEMD160':
+      return new RIPEMD160()
```
Comment (dcousens) [inline browser.js]: this could just be a fall-through,  maybe prefer the `RIPEMD160` branch first

**Comment by dcousens** [comment]
utACK 58e06c1,  obviously blocked on sha.js updating etc

File: browser.js (hunk 1 above)
Comment (fanatid) [inline browser.js]: you're right, updated

**Comment by dcousens** [comment]
is this still current?

**Comment by fanatid** [comment]
`sha.js` still not uses `hash-base`, so still WIP

**Comment by dcousens** [comment]
Close in favour of https://github.com/crypto-browserify/createHash/pull/19

**Pull request closed without merging**